.github/workflows/format.yml

```diff
@@ -0,0 +1,50 @@
+name: Fix
+
+on:
+  push:
```

Comment:
add `pull_request:`

Reply:
I actually don't trigger this workflow for `pull_request` events intentionally, and that's also the reason why I wrote this workflow in a separate YAML file instead of existed `ci.yml`, as I want a different workflow trigger for this one. 


For pull request events, GitHub actions won't run workflows on the development branch. It would first try to merge the development branch and the main branch, then run workflows on that merged commit. This means (a) we cannot run `git push` directly, as the upstream if not the development branch, and (b) if the main branch has some newer commit, pushing the formatted code to the development branch would also include these new content. Because of these two reasons, It seems adding `pull_request` is not a good idea.

My current implementation (i.e. run workflow on the `push` event) can already fix the PR. This PR itself is an example. See the commit history below:

<img width="470" alt="image" src="https://github.com/egoist/tsup/assets/24715727/3b18aba2-3e6d-4b78-9fa5-8e12d969dc74">